pkg/cloudprovider/utils.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cloudprovider
+
+import "k8s.io/kubernetes/pkg/util/sets"
+
+// IsAllowAll checks whether the sets.IPNet allows traffic from 0.0.0.0/0
+func IsAllowAll(ipnets sets.IPNet) bool {
```

Comment:
Moving it.

I'd rather keep it as a function if that's OK, because I'm imagining IPv6 support not being too far down the road...  AWS allows IPv6 on ELBs (but not natively on their internal network other than through tunnelling) - it's much easier to add than making the whole internal network IPv6 (much to my dismay).